htdocs/compta/paiement/list.php

```diff
@@ -211,10 +209,15 @@
 	}
 
 	// Search criteria
-	$sql .= dolSqlDateFilter("p.datep", $day, $month, $year);
 	if ($search_ref) {
 		$sql .= natural_search('p.ref', $search_ref);
 	}
+	if ($search_date_start) {
```

Comment:
We must always keep the detail of the 3 component (day, month, year) for search component so we shoul dhave
if ($search_date_startday) {

if ($search_date_startmonth) {
..